### PR TITLE
Tap: Added warning about vclick

### PR DESCRIPTION
**Pull request by agcolom**
Fixes gh-132

### CODE REVIEW

File: entries/tap.xml [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
 		<xi:include href="../includes/core-extension-desc.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 
+		<div class="warning"><h4>Warning: Use tap with caution</h4>
+			<p>Tap makes use of vclick and therefore should be used with caution on touch devices. See for the <a href="../vclick/">vclick api documentation</a> for more details</p>
```
Comment (arthurvr) [inline entries/tap.xml]: Misses a period, `api` => `API`, and maybe we should change

> See for the vclick api documentation for more details

to 

> See the vclick api documentation for more details

Comment (agcolom) [inline entries/tap.xml]: oh, yes, I changed the text but forgot to remove the beginning! Thanks

**Comment by arthurvr** [comment]
Looks good to me.

**Comment by arschmitz** [comment]
:+1:

**Pull request closed without merging**